html-block-and-inline/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Block and Inline</title>
+</head>
+<body>
+  <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+      <p>Price: <del>$19.99</del> $14.99 </p>
+      <p>Part of our new Americana collection, this <em>beautiful</em> blue is already a customer favorite!</p>
+      <p>Enjoy complete <strong>Freedom of Movement!</strong></p>
+      <p>Does not slip and slide like a regular blanket and keeps you warm from head to toe.</p>
```

Comment:
A little extra indenting here, the `<h2>` and `<p>` tags should all be at the same level.

Reply:
Okay got it, thank you!